terraform/trial_rg/services.tf

```diff
@@ -30,8 +30,9 @@ module "trial_app_service_practitioner" {
   # todo use private endpoint
   settings = {
     "SPRING_APPLICATION_NAME"     = "practitioner-service",
-    "SPRING_CLOUD_CONFIG_PROFILE" = "default",
-    "SPRING_CLOUD_CONFIG_LABEL"   = "main",
+    "SPRING_PROFILES_ACTIVE"      = var.sc_profile,
```

Comment:
"SPRING_PROFILES_ACTIVE" - What does this var means? Why do we need both?

Reply:
this one says which profile THIS service starts with.
and the second one says which config server profile to go to.
in other words:
when I set the active profile on the practitioner it affects the practitioner loaded files.
when I set the config profile it changes to which profile in thhe config server the practitioner asks for

Reply:
fixed. now only 1 var is used: SPRING_PROFILES_ACTIVE